include/revng/Model/FunctionEdgeBase.h

```diff
@@ -35,6 +35,7 @@ type: struct
     doc: Type of the CFG edge
     type: FunctionEdgeType::Values
```

Comment:
Before, we used to identify the class based by looking at the value of `Type` (i.e., certain values were mapped to `FunctionEdge`, certain others to `CallEdge`). I'm not a fan of this newly introduced redundancy, but the CFG (including `FunctionEdgeBase` and friends) are going out of the model, so they won't be user-facing API.
So no biggie.